manifests/README.md

```diff
@@ -0,0 +1,25 @@
+# Argo Events Install Manifests
```

Comment:
Can we add this info under https://argoproj.github.io/argo-events/installation/ as well?

Reply:
Added :+1: I've also reworded the setup for both cluster-wide and namespace install and added a section with detail Kustomize setup. Lmk if there's anything to be changed or reworded.